pkg/operation/botanist/component/logging/fluentoperator/fluent_operator.go

```diff
@@ -0,0 +1,501 @@
+					},
+					Spec: corev1.PodSpec{
+						ServiceAccountName: name,
+						PriorityClassName:  f.values.OperatorPriorityClassName,
+						Containers: []corev1.Container{
+							{
+								Name:            name,
+								Image:           f.values.OperatorImage,
+								ImagePullPolicy: corev1.PullIfNotPresent,
+								Env:             f.computeEnv(),
+								Resources: corev1.ResourceRequirements{
+									Requests: corev1.ResourceList{
+										corev1.ResourceCPU:    resource.MustParse("20m"),
+										corev1.ResourceMemory: resource.MustParse("50Mi"),
+									},
+								},
+								VolumeMounts: f.computeVolumes(),
+							},
+						},
+						Volumes: []corev1.Volume{
+							{
+								Name: "env",
+								VolumeSource: corev1.VolumeSource{
+									EmptyDir: &corev1.EmptyDirVolumeSource{},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+		vpaUpdateMode = vpaautoscalingv1.UpdateModeAuto
+		vpa           = &vpaautoscalingv1.VerticalPodAutoscaler{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name + "-vpa",
+				Namespace: f.namespace,
+			},
+			Spec: vpaautoscalingv1.VerticalPodAutoscalerSpec{
+				TargetRef: &autoscalingv1.CrossVersionObjectReference{
+					APIVersion: appsv1.SchemeGroupVersion.String(),
+					Kind:       "Deployment",
+					Name:       deployment.Name,
+				},
+				UpdatePolicy: &vpaautoscalingv1.PodUpdatePolicy{
+					UpdateMode: &vpaUpdateMode,
+				},
+				ResourcePolicy: &vpaautoscalingv1.PodResourcePolicy{
+					ContainerPolicies: []vpaautoscalingv1.ContainerResourcePolicy{
+						{
+							ContainerName: vpaautoscalingv1.DefaultContainerResourcePolicy,
+							MinAllowed: corev1.ResourceList{
+								corev1.ResourceCPU:    resource.MustParse("100m"),
+								corev1.ResourceMemory: resource.MustParse("128Mi"),
+							},
+						},
+					},
+				},
+			},
+		}
+
+		netpol = &networkingv1.NetworkPolicy{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "allow-fluentbit",
+				Namespace: f.namespace,
+				Annotations: map[string]string{
+					"gardener.cloud/description": "Allows Egress from fluentbit to loki",
+				},
+			},
+			Spec: networkingv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: f.getFBLabels(),
+				},
+				Egress: []networkingv1.NetworkPolicyEgressRule{
+					{
+						To: []networkingv1.NetworkPolicyPeer{
+							{
+								NamespaceSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{
+										v1beta1constants.GardenRole: v1beta1constants.GardenRoleShoot,
+									},
+								},
+								PodSelector: &metav1.LabelSelector{MatchLabels: f.getLokiLabels()},
+							},
+							{PodSelector: &metav1.LabelSelector{MatchLabels: f.getLokiLabels()}},
+						},
+						Ports: []networkingv1.NetworkPolicyPort{
+							{Protocol: &protocolTCP, Port: &lokiPort},
+						},
+					},
+				},
+				PolicyTypes: []networkingv1.PolicyType{networkingv1.PolicyTypeEgress},
+			},
+		}
+
+		config = &corev1.ConfigMap{
```

Comment:
Make this configmap unique.

Reply:
Do you mean to add the data hash in its name and set it as immutable?